main.py

```diff
@@ -31,17 +32,52 @@
 app = FastAPI()
 origins = ['*']
 
+# Define metrcis
+latencies = []
+answer_latencies = []
+add_latencies = []
+drop_latencies = []
+
+requests_total = Counter('requests_total', 'Cumulative requests')
+requests_success_total = Counter('requests_success_total', 'Cumulative successful requests')
+requests_failed_total = Counter('requests_failed_total', 'Cumulative failed requests')
+
+endpoint_requests_total = Counter('endpoint_requests_total', 'Cumulative requests of each endpoint', ['endpoint'])
+endpoint_requests_success_total = Counter('endpoint_requests_success_total', 'Cumulative successful requests of each endpoint', ['endpoint'])
+endpoint_requests_failed_total = Counter('endpoint_requests_failed_total', 'Cumulative failed requests of each endpoint', ['endpoint'])
+
+process_latency_seconds = Histogram('process_latency_seconds', 'Request process latency histogram')
+endpoint_process_latency_seconds = Histogram('endpoint_process_latency_seconds', 'Request process latency histogram of each endpoint', ['endpoint'])
+
+average_process_latency_seconds = Gauge('average_process_latency_seconds', 'Average request process latency')
+endpoint_average_process_latency_seconds = Gauge(
+    'endpoint_average_process_latency_seconds', 'Average Request process latency of each endpoint', ['endpoint']
+)
+
+
 @app.get('/')
 def check_api():
```

Comment:
We should access Prometheus in a unified way, instead of calling prometheus_client for each interface.